packages/amplication-client/src/Resource/resourcePackages/InstalledPackages.tsx

```diff
@@ -0,0 +1,44 @@
+import { List, Snackbar, TabContentTitle } from "@amplication/ui/design-system";
+import React from "react";
+import { match } from "react-router-dom";
+import { AppRouteProps } from "../../routes/routesUtil";
+import { formatError } from "../../util/error";
+import { EnumImages } from "../../Components/SvgThemeImage";
+import { EmptyState } from "../../Components/EmptyState";
+import usePackage from "../hooks/usePackage";
+import { PackageListItem } from "./PackageListItem";
+
+type Props = AppRouteProps & {
+  match: match<{
+    resource: string;
+  }>;
+};
+
+const TITLE = "Installed Packages";
+const SUB_TITLE = "Manage your installed packages";
+
+const InstalledPackages: React.FC<Props> = ({ match }: Props) => {
```

Comment:
```suggestion
const PackagesPage: React.FC<Props> = ({ match }: Props) => {
```

Reply:
also rename the file

Reply:
@morhag90 this comment is still open (the name of the entire component + name of the file itself)

Reply:
@yuval-hazaz @mulygottlieb, we decided to leave it as is for now.